docs/blog/_posts/2021-02-18-scala3-rc1.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog-page
+title: Scala 3.0.0-RC1 – first release candidate is here
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-02-18
+---
+
+Greetings from the Scala 3 team! We are delighted to announce the first release candidate of the stable version of Scala 3 – Scala 3.0.0-RC1.
+
+This release brings some last-minute polishings, clean-ups and changes before the big release. There were a few language changes to improve the user experience, as well as the polishings of the metaprogramming framework. We have also worked on the issues that had to be fixed before the stable release.
+
+Overall, more than [400 PRs](https://github.com/lampepfl/dotty/pulls?q=is%3Apr+is%3Aclosed+closed%3A%3E2020-12-02+sort%3Acomments-desc) were merged after the M3 release and until today! Read more below!
+
+<!--more-->
+
+# New `import` syntax
+The following are the changes to the `import` syntax made in this release.
+
+Wildcard import `_` is replaced by `*`. The motivation is that the majority of other languages use `*`. For example:
+
+```scala
+import scala.annotation.*  // imports everything in the annotation package
+```
+
+Renaming operator `=>` is replaced by a soft keyword `as`. `as` is also allowed outside braces. For example:
+
+```scala
+import scala.collection.mutable as mut
+import NumPy as np
+```
+
+For the details and discussion, see [PR #11244](https://github.com/lampepfl/dotty/pull/11244). Read more about this change in the [documentation](https://dotty.epfl.ch/docs/reference/changed-features/imports.html).
+
+# Use `unitialized` for wildcard initializers
+An obscure use of _ occurs in var definitions:
+
+```scala
+var x: T = _
+```
+
+It defines a concrete variable x without an initial value, or rather the default initial value that the JVM assigns to object fields. It can only be used in a class or object, not to initialize a local variable.
+
+We came up with an arguably better way to express this idiom: the special `uninitialized` value in the `scala.compiletime` object. To get an uninitialized field, you now write:
+
+```scala
+import scala.compiletime.uninitialized
+
+var x: A = uninitialized
+```
+
+This way expresses the intent of the idiom in a more verbose and easy to read way than simply writing an underscore.
+
+For discussion, see [PR #11231](https://github.com/lampepfl/dotty/pull/11231), and the [documentation](https://dotty.epfl.ch/docs/reference/dropped-features/wildcard-init.html) is available on our website.
+
+# Allow `with` after class
+[PR #11157](https://github.com/lampepfl/dotty/pull/11157) brings an syntactic change whereby you have to write `with` instead of `:` to signify the start of the body of a class definition. For example:
```

Comment:
headline says "allow", but body text says "you have to", which is it?

and anyway I thought we had settled on `:` not `with`?

Reply:
@odersky I assume `with` will supersede `:` completely in future? So far you can indeed use both, but I assume the `:` will be deprecated at some point?

Reply:
The decision is to stay with `:`, see #11213.

Reply:
Hm, so why was `with` added if we are sticking with `:`?

Reply:
Here is the final decision post by Martin at the end of the scala contributors gigantic thread, now closed: 
https://contributors.scala-lang.org/t/feedback-sought-optional-braces/4702/588
I guess `with` was added while trying things out, but finally colon showed up as a winner.

Reply:
Yes, we went back to `:`. `with` is not allowed after classes.

Reply:
(The 3.0.0-RC1 release notes on GitHub need updating on this point as well.)

Reply:
This is not the case, you can still compile the following file perfectly fine:
```scala
class Foo with
  def foo = 23

@main def Test = println(Foo().foo)
```

Reply:
This seems to have been made possible by #10861, which was not reverted.